.travis.yml

```diff
@@ -1,79 +0,0 @@
-language: python
```

Comment:
> @amarjandu Instead of doing a fixup, I edited the commit. It made atticcing the file easier. Sorry if that makes it harder to review!

The exploring the commit on GH shows that this file was deleted still 😮

Reply:
<img width="1901" alt="Screen Shot 2021-04-21 at 10 43 11 AM" src="https://user-images.githubusercontent.com/1947534/115597687-621c2700-a28e-11eb-9361-e86c8b093e69.png">